src/main/java/net/atos/zac/app/ontkoppeldedocumenten/converter/RESTOntkoppeldDocumentConverter.java

```diff
@@ -31,6 +31,8 @@ public RESTOntkoppeldDocument convert(final OntkoppeldDocument document) {
         restDocument.ontkoppeldDoor = userConverter.convertUserId(document.getOntkoppeldDoor());
         restDocument.ontkoppeldOp = document.getOntkoppeldOp();
         restDocument.reden = document.getReden();
+        restDocument.isVergrendeld = document.getEnkelvoudigInformatieObjectLock() != null && document.getEnkelvoudigInformatieObjectLock()
```

Comment:
I think we can get `enkelvoudigInformatieobject` and fetch the lock via it. We do this in `PolicyService` like that:
```
lockService.findLock(parseUUIDFromResourceURI(enkelvoudigInformatieobject.getUrl()))
```

This means we won't need changes in the document model. @edgarvonk WDYT?

Reply:
I chose to do it this way because I think the performance will be better if we join the two tables. But I'm fine with using the lockService too.

Reply:
I don't really have a preference to be honest. Would be good to have only one way to get a document lock in our code I think. I would not worry about performance too much here for now.

Reply:
Allright, I will use the service :-)

Reply:
Done